docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Metrics and targets"
```

Comment:
Do we really want to mix metrics and targets here and then pick targets up again in the concurrency (and RPS) chapter?